### DISCUSSION TITLE
How do you enter command line arguments? (SOLVED)

### DISCUSSION THREAD

**Discussion by Lowen-Beehold** [comfyanonymous/ComfyUI#3158] [Q&A]
This has been driving me crazy trying to figure it out .

on the front page it says  "Use --preview-method auto to enable previews." so I spent ages online looking how/where I enter this command.  found this post on reddit - 
https://www.reddit.com/r/comfyui/comments/15jxydu/comfyui_command_line_arguments_informational/ 
that says to "cd into your comfy directory ; run python main.py -h" so I tried running main.py with python but the is still no option to enter this command. I feel like I'm either being a simpleton or missing a key piece of information but either way can someone please put me out of my misery because I have spent a couple of hours already looking for answers.

**Comment by gtx155** [comment]
Command line arguments can be put in the bat files used to run comfyui like this separated by a space after each command
![run](https://github.com/comfyanonymous/ComfyUI/assets/27731920/952c9134-e4db-48c3-98f9-74c77395a3e3)

**Reply by Lowen-Beehold** [reply]
THANK YOU! I knew it was going to be something simple.
